File: JavaApp/test/app/src/main/java/com/example/test/MainActivity.java
package com.example.test;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.widget.FrameLayout;

public class MainActivity extends AppCompatActivity {

    FrameLayout frameLayout;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        frameLayout = (FrameLayout) findViewById(R.id.frame);
        frameLayout.setBackgroundResource(R.color.teal_200);
    }
}

File: KotlinApp/SplashScreenExample/app/src/main/java/com/saehyun/splashscreenexample/MainActivity.kt
package com.saehyun.splashscreenexample

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        setTheme(R.style.Theme_SplashScreenExample)
        super.onCreate(savedInstanceState)

        setContentView(R.layout.activity_main)
    }
}